os/board/artik05x/userspace/Makefile

```diff
@@ -94,6 +94,7 @@ OBJS  = $(COBJS)
 # Targets:
 
 all: $(USER_BINPATH)$(DELIM)tinyara_user.elf $(USER_BINPATH)$(DELIM)User.map
+	$(call DELFILE, tinyara_user.elf)
```

Comment:
1. `$(USER_BINPATH)` is needed.
2. That file will be deleted when we call `CLEAN`. Is this necessary to delete here? Then, Why do you need to support deleting it in CLEAN?

Reply:
After make, if we do git status , it shows `board/artik05x/userspace/tinyara_user.elf`
Its not good to show these unwanted build artifacts. 
Either hide these by adding it to `.gitignore` OR delete. 
Since this elf already moved to `build/output/bin/`, it is no more useful, hence deleted the file.

Yes, if it is deleted after the build, no need to do the same in clean. I will remove it from clean target